packages/react-components/src/atoms/__tests__/Dropdown.test.tsx

```diff
@@ -151,3 +156,98 @@ it('gets greyed out when disabled', () => {
   ).not.toBe(silver.rgb);
   expect(getByRole('textbox')).not.toBeDisabled();
 });
+
+it('shows the field in red when required field not filled', async () => {
+  const { getByText, getByRole, rerender } = render(
+    <Dropdown
+      options={[{ value: 'LHR', label: 'Heathrow' }]}
+      required={true}
+      value=""
+    />,
+  );
+
+  userEvent.click(getByText('Select'));
```

Comment:
Please put the placeholder into the props. Changing the default placeholder shouldn't break the tests